egs2/ami/asr1/run.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#  Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+stage=0
+stop_stage=10
+nj=32
+
```

Comment:
You don't need to specify stage, stop_stage, nj  because they are not task depended.
I'd like to make all recipes to follow asr.sh as possible.